TagManager.php

```diff
@@ -67,7 +67,8 @@ public function registerEvents()
             'Template.bodyClass' => 'addBodyClass',
             'Access.Capability.addCapabilities' => 'addCapabilities',
             'TwoFactorAuth.requiresTwoFactorAuthentication' => 'requiresTwoFactorAuthentication',
-            'Db.getTablesInstalled' => 'getTablesInstalled'
+            'Db.getTablesInstalled' => 'getTablesInstalled',
+            'Template.embedReactTagManagerTrackingCode' => 'embedReactTagManagerTrackingCode',
```

Comment:
I don't see this being used. Is it so that Cloud can override the behaviour?

Reply:
no we are calling it from core this is how we get the code in Matomo Tag manager tab, so followed similar way here